__tests__/integration/helpers/testRunner.js

```diff
@@ -95,7 +95,7 @@ const setUpTestRunner = async ({
   }));
 
   // uncomment this line in order to view full logs for debugging
-  // logWatcherDuplex.pipe(process.stdout);
+  logWatcherDuplex.pipe(process.stdout);
```

Comment:
```suggestion
 //  logWatcherDuplex.pipe(process.stdout);
```

Reply:
I'm personally keen on keeping this like this.

When would you be looking at the integration test logs, and not want the pod logs right there?